packages/server-wallet/src/models/channel.ts

```diff
@@ -152,7 +152,7 @@ export class Channel extends Model implements RequiredColumns {
   ): Promise<Channel[]> {
     return Channel.query(txOrKnex)
       .select()
-      .where({assetHolderAddress, participants});
+      .where({assetHolderAddress, participants: JSON.stringify(participants)});
```

Comment:
How does this approach compare to [this approach](https://github.com/statechannels/statechannels/pull/2792/commits/4ec1b65185bc8748fffa29e3de1f0317878f74a2)?

Reply:
I don't think that approach works and that issue is hidden since only `[]` is passed in on that PR

Reply:
@kerzhner

Reply:
I am not 100% sure what "this approach doesn't work" means in this case :/ My observations are:
1. I don't know what the implications of `JSON.stringify` are. Is some/all validation skipped for the participants column?
1. For the `transferredOut` column, it is **not** the case that `[]` is the only value used: https://github.com/statechannels/statechannels/blob/asset-transfer/packages/server-wallet/src/models/funding.ts#L93

Reply:
I mean that when I add
```
participants: {
  type: 'array',
  items: {type: 'object'},
}
```
in the same way as `transferredOut` above and run the test I wrote, I get the above `DataError`

Reply:
It may be the case that "you're doing it wrong", and objection + knex makes it difficult to know what you are doing wrong, and what is the correct thing to do.

Reply:
If so, I was unable to figure out how to do it "right"

Reply:
So leaving be the issue of whether or not this change is required, and coming back to
> I don't know what the implications of JSON.stringify are. Is some/all validation skipped for the participants column?

Do you know what the implications are? If there are no implications, then we might not need to understand the "right" way to do it.

Reply:
I thought about this for a few minutes and created [this](https://github.com/statechannels/statechannels/pull/2812) to continue the discussion